mlx/fast.cpp

```diff
@@ -87,7 +87,7 @@ array rms_norm(
   if (s.device == Device::gpu) {
     return array(
         x.shape(),
-        x.dtype(),
+        out_type,
```

Comment:
that looks like a bug.. 😬

Reply:
Looks like a bug now or before? Cause I thought I was fixing a bug :-)

Reply:
Yea I meant before. Good catch!